config/helm/config.yaml

```diff
@@ -0,0 +1,2 @@
+leader-elect: {{ .Values.leaderElect }}
+metrics-bind-address: {{ .Values.metricsBindAddress | quote }}
```

Comment:
This is the content of the CM. Grep `config/helm/kustomization.yaml` file for `configMapGenerator `